packages/selinux-policy/fs.cil

```diff
@@ -45,6 +45,8 @@
 (filecon "/.*/usr/bin/migrator" file api_exec)
 (filecon "/.*/usr/bin/storewolf" file api_exec)
 (filecon "/.*/usr/bin/cfsignal" file api_exec)
+(filecon "/.*/usr/bin/prairiedog" file api_exec)
+(filecon "/.*/usr/bin/thar-be-settings" file api_exec)
```

Comment:
`thar-be-settings` is ordinarily invoked by `apiserver`, which runs as `api_t`, but it's also invoked as a standalone binary in startup, where it runs as the default `system_t`. Transition rule makes sense - one of the settings it's applying can now trigger a write to the datastore filesystem.

`prairiedog` is invoked in two contexts:
* as a setting generator by sundog; it will have the `system_t` label, but it shouldn't need to write to the datastore filesystem here, since it's just adding settings via API
* as a restart command; it should inherit the `api_t` label since it's spawned by `apiserver`, which should allow it to write to the datastore filesystem.

So I'm not sure why `prairiedog` needs the label.